config/locales/en/prior_authority/applications.yml

```diff
@@ -1,5 +1,38 @@
 en:
   prior_authority:
+    adjustments:
```

Comment:
I think this stuff maybe ought to be in a new file `config/locales/en/prior_authority/adjustments.yml`. At some point (possibly a dev time you missed) we discussed having a larger number of smaller yml files rather than one big monster one.

Reply:
done